stonk_market_mayhem.py

```
@@ -0,0 +1,76 @@
+def sell_stock():
+    global money
+    stock = input("Which stock do you want to sell? ")
+    qty = int(input("How many shares? "))
+    if stock in portfolio and portfolio[stock] >= qty:
+        money += stocks[stock] * qty
+        portfolio[stock] -= qty
+        print(f"Sold {qty} shares of {stock}")
+    else:
+        print("You don’t own that much.")
+
+def view_portfolio():
```

Ruff (0.8.2)

Missing return type annotation for public function `view_portfolio`

Add return type annotation: `None`

(ANN201)